sensors:
  imu:
    node: bno08x
    imu_topic: sensors/bno08x/raw
    mag_topic: sensors/bno08x/mag
    sample_rate: 400

  distance:
    node: vl53l0x
    topic: sensors/vl53l0x/dist
    sample_rate: 400

  color:
    node: tcs34725
    topic: sensors/tcs34725/color
    sample_rate: 100

  camera:
    node: picamera
    topic: sensors/picamera/compressed
    sample_rate: 10 # is also framerate
    input_stream: 0
    resolution:
      width: 640
      height: 480

  battery:
    node: ina219
    topic: sensors/ina219/voltage
    sample_rate: 0.5
    max_cell_voltage: 4.2
    min_cell_voltage: 3.8
    cells: 2
    capacity: 1800 # mAh

led:
  mark_color_upper: [22, 22, 0]
  mark_color_lower: [30, 30, 5]
  deactivation_delay: 0.5

imu_tracker:
  node: imu_tracker
  topic: imu_tracker/pose
  sample_rate: 400
